tests/unit/amo/components/TestApp.js

```diff
@@ -119,13 +114,6 @@ describe(__filename, () => {
     });
   });
 
-  it('renders an ErrorPage component', () => {
```

Comment:
You can put this test back now

Reply:
The test was checking that we could pass in a custom `ErrorPage` component, but that was never used anywhere other than this test, so I removed it, much like I removed `HeaderComponent` and `FooterComponent`. Without the ability to override `ErrorPage`, this test seems pointless. WDYT?

Reply:
Oh, whoops, I see. My best guess is that it was added to prove dependency injection before we could do shallow rendering. Sure, you can remove it. I see that dependency injection is longer possible, which is fine: https://github.com/mozilla/addons-frontend/pull/8717/files#diff-53db53641895141db51830eafab741bcL37

Reply:
Correct. I removed that "feature".